logger.js

```diff
@@ -3,7 +3,7 @@ path = require("path"),
 debugging = process.argv.slice(1).join(" ").indexOf("debug") > -1,
 debug = (debugging ? (msg)=>{console.log(msg);} : ()=>{});
 
-module.exports = (externalLogger, logFolder, logFilePrefix = "installer")=> {
+module.exports = (externalLogger, logFolder, from, logFilePrefix = "installer")=> {
```

Comment:
@jungeunyoon In looking at this again, `from` and `logFilePrefix` could very well end up being the same value. Lets change it to `moduleName` and ensure it's used throughout the rest of the file.
`(externalLogger, logFolder, moduleName = "installer")`

Reply:
@stulees Should be good~